clients/pkg/promtail/scrapeconfig/scrapeconfig.go

```diff
@@ -41,6 +41,7 @@ type Config struct {
 	PushConfig        *PushTargetConfig          `mapstructure:"loki_push_api,omitempty" yaml:"loki_push_api,omitempty"`
 	WindowsConfig     *WindowsEventsTargetConfig `mapstructure:"windows_events,omitempty" yaml:"windows_events,omitempty"`
 	KafkaConfig       *KafkaTargetConfig         `mapstructure:"kafka,omitempty" yaml:"kafka,omitempty"`
+	AzurelogConfig    *AzurelogTargetConfig      `mapstructure:"azurelog,omitempty" yaml:"azurelog,omitempty"`
```

Comment:
Based on, https://github.com/grafana/loki/pull/8787#discussion_r1138548299, I'm wondering if this should be AzurelogConfig or AzureEventHubConfig. We are building something which works with Azure Event Hub and isn't constrained to just azure resource logs from event hub.

Reply:
You are right 👍 I will change it. Also, I will rename a package and the docs section